src/state/slices/stakingDataSlice/selectors.ts

```diff
@@ -42,6 +42,18 @@ export const selectStakingDataByAccountSpecifier = createSelector(
   }
 )
 
+export const selectTotalStakingDelegationCryptoByAccountSpecifier = createSelector(
+  selectStakingDataByAccountSpecifier,
+  stakingData => {
+    // We make the assumption that all delegation rewards come from a single denom (asset)
+    // In the future there may be chains that support rewards in multiple denoms and this will need to be parsed differently
+    return stakingData?.delegations?.reduce(
```

Comment:
Note: When @cjthompson asset-service metadata changes will be ready, this could be surfaced directly from the asset itself

Reply:
@gomesalexandre what are you referring to here?

Reply:
We had previously discussed about asset-service potentially handling denoms as metadata. For now however, we will go with web not having knowledge of denoms at all and just using `assetId` as a discriminator, as we have currently